bitnami/postgresql-repmgr/12/debian-11/rootfs/opt/bitnami/scripts/librepmgr.sh

```diff
@@ -649,7 +665,7 @@ repmgr_rewind() {
     info "Rejoining node..."
 
     ensure_dir_exists "$POSTGRESQL_DATA_DIR"
-    if is_boolean_yes "$REPMGR_USE_PGREWIND"; then
+    if is_boolean_yes "$REPMGR_USE_PGREWIND" && ! is_boolean_yes "$REPMGR_SWITCH_ROLE"; then
```

Comment:
I am not sure about this condition, Why aren't you running `pg_rewind` when the role is switching?

Reply:
That is something I need to find out and is not clear to me. I learned the hard way a failed primary cannot be rewinded and needs some extra care. Something for another pull request.